src/errors.c

```diff
@@ -141,12 +141,11 @@ void Error_InvalidFilterPlacement(rax *entitiesRax) {
 void Error_SITypeMismatch(SIValue received, SIType expected) {
 	size_t bufferLen = 256;
 	size_t bytesWritten = 0;
-	char *buf = rm_malloc(bufferLen * sizeof(char));
+	char buf[256];
```

Comment:
Why not just this?
```c
char buf[bufferLen];
```
Or better yet, perhaps we could remove the `size_t bufferLen = 256;` and define a named constant as this magic number is used everywhere within the file?

Reply:
Thanks. I made changes to use a named constant.

Reply:
Great, thanks!